geode-serialization/src/main/java/org/apache/geode/internal/serialization/SerializationSanctionedSerializablesService.java

```diff
@@ -12,15 +12,14 @@
  * or implied. See the License for the specific language governing permissions and limitations under
  * the License.
  */
-package org.apache.geode.distributed.internal;
+package org.apache.geode.internal.serialization;
 
-import java.io.IOException;
-import java.util.Collection;
+import java.net.URL;
 
-public interface DistributedSystemService {
-  void init(InternalDistributedSystem internalDistributedSystem);
+public class SerializationSanctionedSerializablesService implements SanctionedSerializablesService {
```

Comment:
Note: github diffs are confused... the code is correct.

DistributedSystemService was renamed to SanctionedSerializablesService and moved to geode-serialization.

SerializationSanctionedSerializablesService is a new class that implements SanctionedSerializablesService and also lives within geode-serialization.